plugins/imjournal/imjournal.c

```diff
@@ -520,24 +520,23 @@ persistJournalState(void)
 	DEFiRet;
 	FILE *sf = NULL; /* state file */
 	char tmp_sf[MAXFNAME];
-	size_t n;
 
 	DBGPRINTF("Persisting journal position, cursor: %s, at head? %d\n",
 			  journalContext.cursor, journalContext.atHead);
 
 	/* first check that we have valid cursor */
 	if (!journalContext.cursor) {
+		DBGPRINTF("Journal cursor is not valid, ok...\n");
 		ABORT_FINALIZE(RS_RET_OK);
 	}
 
 	/* we create a temporary name by adding a ".tmp"
 	 * suffix to the end of our state file's name
 	 */
-	n = snprintf(tmp_sf, sizeof(tmp_sf), "%s.tmp", cs.stateFile);
-	if (n >= sizeof(tmp_sf)) { /* backup in case of too long path name */
-		strncpy(tmp_sf, cs.stateFile, sizeof(tmp_sf) - 5);
-		strcat(tmp_sf, ".tmp");
-	}
+#define SUFFIX ".tmp"
```

Comment:
maybe a nitpick, however `SUFFIX` seems quite generic, I would be afraid of it clashing with something.

Reply:
Agreed, I will fix it.